### ISSUE TITLE
Not working w. protractor 4

### ISSUE THREAD

**Issue by mdix** [Updater/protractor-console#6]
We're using protractor `4.0.14` in our project and protractor-console doesn't seem to work with it. I installed the package as described and added it to our `protractor.conf.js`:

```javascript
exports.config = {
    plugins: [{
        package: 'protractor-console',
        logLevels: ['info'],
    }],
    seleniumAddress: 'http://localhost:4444/wd/hub'
```

Logs don't show up. I set the logLevel to 'debug' and also tried 'info' but nothing is displayed. 

Might be related to the last part of the first sentence in the protractor [3.0.0 CHANGELOG](https://github.com/angular/protractor/blob/master/CHANGELOG.md#300):

> and plugins now need to be explicitly required

**Comment by pmowrer** [comment]
Have you successfully used `protractor-console` before with another version of Protractor?

If you're looking specifically for `info` and `debug` level logs, it could be that they're suppressed. Please see: https://github.com/Updater/protractor-console/issues/4

**Comment by mdix** [comment]
No. My first time using `protractor-console`. I added a `console.error` to the code and also tried w. a `throw`, nothing changed.

BTW: Thanks for your help.

**Comment by pmowrer** [comment]
FWIW, it's working for us and we've been on Protractor `4.0.10` for some time.

What browser(s) have you tried?

**Comment by mdix** [comment]
Just Google Chrome.

**Comment by aperdomob** [comment]
what happen with this issue?
